src/analysis/inner_product_space/adjoint.lean

```diff
@@ -186,6 +186,29 @@ lemma is_adjoint_pair (A : E' →L[ℝ] F') :
 
 end real
 
+section unitary
+
+variables {U : (E →L[𝕜] E)}
+
+/-- Builds a linear isometric equivalence from `U ∈ unitary (E →L[𝕜] E)`. -/
+def linear_isometry_equiv_of_unitary (hU : U ∈ unitary (E →L[𝕜] E)) : (E ≃ₗᵢ[𝕜] E) :=
```

Comment:
Does this actually require continuity?

Reply:
Ah no, the same story goes through nearly identically in the finite dimensional case.  I've added this.